terraform/20-app/main.tf

```diff
@@ -28,3 +35,53 @@ terraform {
     key            = "app/state.tfstate"
   }
 }
+
+module "cognito" {
+  source = "../modules/cognito"
+
+  user_pool_name    = "app-${terraform.workspace}-user-pool"
+  client_name       = "app-${terraform.workspace}-client"
+  user_pool_domain  = "app-${terraform.workspace}-auth"
+
+  callback_urls = ["https://${terraform.workspace}.dev.ukhsa-dashboard.data.gov.uk/callback"]
```

Comment:
We'll have to tweak the structure of this depending on where we've deplpoyed this env. But logically this makes sense